<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotationWidget</class>
 <widget class="QWidget" name="RotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rotations in degrees</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="xAngle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QPushButton" name="rotateButton">
        <property name="text">
         <string>Rotate..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="zAngle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="yAngle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Workspace.qrc"/>
 </resources>
 <connections/>
</ui>
